app/scripts/metamask-controller.js

```diff
@@ -48,6 +43,12 @@ const seedPhraseVerifier = require('./lib/seed-phrase-verifier')
 const cleanErrorStack = require('./lib/cleanErrorStack')
 const log = require('loglevel')
 
+const AddressBookController = require('gaba/AddressBookController').default
```

Comment:
This import syntax is ugly for now because GABA outputs ES modules (technically UMD) but they're being required here as CJS modules. We will eventually use pure ES module syntax everywhere so this will change to:

```js
import {
    AddressBookController,
    CurrencyRateController,
    NetworkStatusController,
    PhishingController,
    TokenRatesController
} from 'gaba';
```